tests/foreman/sys/test_fam.py

```diff
@@ -166,7 +166,7 @@ def test_positive_ansible_modules_installation(target_sat):
         doc_name = result.stdout.split('\n')[1].lstrip()[:-1]
         assert doc_name == module_name
     # check installed modules against the expected list
-    assert FOREMAN_ANSIBLE_MODULES.sort() == installed_modules.sort()
+    assert sorted(FOREMAN_ANSIBLE_MODULES) == sorted(installed_modules)
```

Comment:
because of this, we never noticed that the lists need updating :crying_cat_face: